lib/src/utils/constants.dart

```diff
@@ -1 +1,6 @@
+import 'package:flutter/material.dart';
 
+class Platform {
```

Comment:
@euferreira Why not use the Platform class form dart:io?

Reply:
Please re-evaluate the snippet.
I had created this platform helper to assist in the implementation of unit tests, but it turned out to be unnecessary.